p2p/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name       = "p2p"
```

Comment:
Should we not maybe call this `tendermint-p2p`? That'd fit with the naming convention of the other crates. Plus there's already a [p2p crate](https://crates.io/crates/p2p).

`p2p` is fine though for the directory name, of course.